src/app/components/auth/login/login.component.ts

```diff
@@ -91,25 +70,50 @@ export class LoginComponent implements OnInit, AfterViewInit {
     self.router.navigate([redirectTo]);
   }
 
-  /**
-   * Submit form function. Called after validation
-   * @param self  context value of this
-   */
-  formSubmit(self) {
-    const LOGIN_BODY = JSON.stringify({
-      username: self.globalService.formValueForLabel(self.ALL_FORMS[self.loginForm], 'username'),
-      password: self.globalService.formValueForLabel(self.ALL_FORMS[self.loginForm], 'password')
-    });
-    self.apiService.postUrl(self.endpointsService.loginURL(), LOGIN_BODY).subscribe(
+
+  // Function to login
+  userLogin(loginFormValid) {
+    if (!loginFormValid) {
+      this.globalService.stopLoader();
+      return;
+    }
+
+    this.globalService.startLoader('Taking you to EvalAI!');
```

Comment:
Does this appear? I think I didn't see it when I had tested it.

Reply:
yes it was there